operator/src/test/java/org/keycloak/operator/testsuite/unit/IngressLogicTest.java

```diff
@@ -174,4 +181,15 @@ public void testHttpSpecWithoutTlsSecret() {
         assertEquals("HTTP", reconciled.get().getMetadata().getAnnotations().get("nginx.ingress.kubernetes.io/backend-protocol"));
         assertEquals("edge", reconciled.get().getMetadata().getAnnotations().get("route.openshift.io/termination"));
     }
+
+    @Test
+    public void testCustomAnnotations() {
```

Comment:
Could you also please add test for removing a previously set annotation?

Reply:
@vmuzikar added tests in `KeycloakIngressTest` because I'm not sure if the tests I wrote in this class are updating or just creating a new Ingress from scratch.

Reply:
@pruivo Thanks for the update.

To properly test that existing annotations are updated, you need to update:
https://github.com/keycloak/keycloak/blob/4741af7a40a7350bd3c4699c3e7728f93a8b6cf6/operator/src/test/java/org/keycloak/operator/testsuite/unit/IngressLogicTest.java#L105-L111
Please add some existing dummy annotations here.

Reply:
ok, I think I got it now :+1: 
PR is updated.